CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,5 @@
+- Fix: explicitAttrs of device was tainted even if not defined
+- Fix: do not include static, lazy and commands from group to device to avoid duplicate them in device (#1377) 
```

Comment:
I think the following paragraph in documentation in doc/api.md

> If devices are not pre-registered, they will be automatically created when a measure arrives to the IoT Agent - this
process is known as autoprovisioning. The IoT Agent will create an empty device with the group `apiKey` and `type` - the associated document created in database doesn't include config group parameters (`timestamp` and `explicitAttrs` in particular). The IoT Agent will also create the entity in the Context Broker if it does not exist yet.

should mention the elements this PRs add to the exclusion, I mean change this part to:

> (in particular, `timestamp`, `explicitAttrs`, static and lazy attributes and commands)

Reply:
> should mention the elements this PRs add to the exclusion, I mean change this part to:
> 
> > (in particular, `timestamp`, `explicitAttrs`, static and lazy attributes and commands)

At this point, would also make sense to include `active` (or attributes in IoTA):

> (in particular, `timestamp`, `explicitAttrs`, `active` ir `attributes`, `static` and `lazy` attributes and commands)

Reply:
Let's use @mapedraza 's version

Reply:
Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1396/commits/e9af104cf73c7a707a6f00d50cde213545f39df6